### PR TITLE
App new urls

### PR DESCRIPTION
**Pull request by ArtemLapkin**

### CODE REVIEW

File: admin/views/view_my_items.php [hunk 1]
```diff
@@ -215,5 +232,25 @@ function loadMore(size, dataLength, item) {
 			}
 		});
 	}
+
+	window.opinoinstageHandleSelectLinks = function(elm){
```
Comment (kucaahbe) [inline admin/views/view_my_items.php:236]: inline js is illegal, let's move it to file and use jquery [document ready](https://api.jquery.com/ready/)

Comment (kucaahbe) [reply admin/views/view_my_items.php:236]: and looks like this function (opinoinstageHandleSelectLinks) is never used

File: admin/views/view_my_items.php [hunk 2]
```diff
@@ -215,5 +232,25 @@ function loadMore(size, dataLength, item) {
...
+		var url = $(elm).children("option:selected").data('href');
+		if( url ) {
+			window.location = url;
+		}
+	}
+
+	if( $('.opinoinstage-show-anchor-list').length > 0 ) {
```
Comment (kucaahbe) [inline admin/views/view_my_items.php:243]: typo? better to fix

File: admin/views/view_my_items.php [hunk 3]
```diff
@@ -215,5 +232,25 @@ function loadMore(size, dataLength, item) {
...
+		$(document).click(function(e){
+			if( $(e.target).is('.opinoinstage-show-anchor-list') ) {
```
Comment (kucaahbe) [inline admin/views/view_my_items.php:245]: it should be:
```javascript
jQuery(function($) {
  $('.opinoinstage-show-anchor-list').on('click', function (event) {
    event.preventDefault()
   $(this).toggleClass('opened')
  })
})
```
(not sure about exact function names, just describing general idea)

Comment (ArtemLapkin) [reply admin/views/view_my_items.php:245]: if I do it this way - it will not close dropdown if user clicks outside of dropdown

**Review by kucaahbe — APPROVED** [review]

**Pull request merged**